lib/json_api_resource/queryable.rb

```diff
@@ -45,7 +45,8 @@ def single_result(results)
 
         result.meta = results.meta
 
-        result.errors = ApiErrors(results.errors)
+        # There will never be errors here but this provides a good default for forms
+        result.errors = ActiveModel::Errors.new(result)
```

Comment:
You are right, it is not required because we do the same defaulting in resource.  I removed it.